### ISSUE TITLE
Minor Accessibility Enchancements

### ISSUE THREAD

**Issue by dennis-varvari** [h2qutc/ngx-mat-datetime-picker#93]
Hi,

When using your datetime picker in a project that has accessibility in mind (e.g. WCAG), I encountered some minor problems, specifically when using screen readers.
It would be so helpful to include some of these small enhancements as they would make a big difference!

Some of the image buttons on the picker have no accessible names or labels, so screen readers don't read the functionality of these buttons to the users:
- the checkmark button that saves the input of the picker has no textual description - you could add an `aria-label="Done"` attribute to the html tag of the button (the screen reader would be able to read the text "Done" to a user whenever they would change focus to the checkmark button) _see the first attachment_

- the time addition and subtraction buttons seem to have identical accessible names that don't reflect their usage, such as 'expand_less icon' and 'expand_more icon' - you could add more descriptive textual descriptions to them such as "Previous Hour", "Next Minute" or so by using the `aria-label` attribute (the user of the screen reader could understand the usage of the button that they placed their focused on) _see the second attachment_

I have noticed that some of the components of the picker already have accessibility labels that screen readers can interpret. It would be great to extend this functionality.

Thank you!

![image](https://user-images.githubusercontent.com/87427657/187528138-1146d51b-63db-4e82-96c4-1779e9a30963.png)
![image](https://user-images.githubusercontent.com/87427657/187528366-ff0eb23a-9603-42ac-9d72-b6071eeb5422.png)
